WordPress/gutenberg
# [RNMobile] Moving native tests to gutenberg repo

`gutenberg-mobile` side PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/977

This PR moves some of the `mobile-gutenberg` test suits to the `gutenberg` repo.
Added an exception to `test/unit/jest.config.js` so these tests are not run by `npm run test-unit`.

As a intermediate step, these tests will continue to be run only from `gutenberg-mobile` checkout via `yarn test`.

On a next step we plan to integrate these mobile tests to the `gutenberg` tool-chain.

To test:
- On a stand-alone checkout of `gutenberg` run `rpm run test-unit`
- Check that none of these new tests are run.
- Check that all tests pass.
